Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/spark/common/SparkBatchJob.java

```diff
@@ -158,6 +162,17 @@ public SparkBatchJob(
         this.ctrlSubject = ctrlSubject;
```

Comment:
Update this with `this(submissionParameter, sparkBatchSubmission, ctrlSubject, null)` please